### PR TITLE
Template improvement

### PR DESCRIPTION
**Pull request by veelenga**
* removed trailing characters
* updated license copyright date

### CODE REVIEW

**Comment by drujensen** [comment]
Crystal is working in the docker image but failing in the travis instance.  We are not dependent on that so we should probably remove the language from the .travis.yml file

**Comment by elorest** [comment]
Lets get this merged?

File: src/templates/app/shard.yml.ecr [hunk 1]
```diff
@@ -10,17 +10,18 @@ license: MIT
 
 dependencies:
   amber:
+
```
Comment (elorest) [inline src/templates/app/shard.yml.ecr]: This extra line shouldn't be here. Tests fail because of it.

**Comment by elorest** [comment]
I'm fine with this but line 13 needs to be removed from shards.yml.ecr so the tests will pass.

**Comment by veelenga** [comment]
Yup, will remove it. BTW, that came from master branch :) https://github.com/Amber-Crystal/amber_cmd/commit/e5e64e3fa9dd4d261ab38b8bec4a13e65d6fb31b. Thanks for review

**Comment by elorest** [comment]
Weird. It didn't break all of the tests here.

**Pull request merged**